.github/workflows/tests.yml

```diff
@@ -44,10 +41,14 @@ jobs:
       - name: Install libgpgme devel package
         run: sudo apt install -y libgpgme-dev libbtrfs-dev libdevmapper-dev
 
+      - name: Extract golangci-lint version from Makefile
+        id: golangci_lint_version
+        run: echo "GOLANGCI_LINT_VERSION=$(awk -F '=' '/^GOLANGCI_LINT_VERSION *=/{print $2}' Makefile)" >> "$GITHUB_OUTPUT"
```

Comment:
Perhaps it would be cleaner, if we want to keep these things aligned, to put them in a nicer machine readable (or just plain text) third place that both places can read.  Perhaps a `versions/` dir with some plain files like `versions/golang-lint-version` with just the number in there that the action and makefile can `cat`.

Also the message for this commit is too long.  Please keep it below 79 chars.

Reply:
which one?
```
git log -4 --pretty=format:"%s

%b"|while read line; do echo "${#line}: ${line}"; done
49: Makefile: include running go tests in `make test`
0: 
0: 
24: bib: fix linter problems
0: 
65: New linter problems arose after updating golangci-lint's version.
0: 
75: Makefile,github: implement `make lint` and centralize GOLANGCI_LINT_VERSION
0: 
0: 
34: github: bump golangci-lint version
```

Reply:
Sorry, it's actually 72.  Miscounted :)